### PR TITLE
DISCO-1222: Documenting why we cannot change Course numbers

### PR DESCRIPTION
**Pull request by Dillon-Dumesnil**

### CODE REVIEW

File: docs/decisions/0006-disallow-course-number-changes-in-publisher.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+Upon Course Run creation, the Course key is used to create the Course Run
+key. The Course Run key is then used as part of the URL in the LMS and in many
+other places.
+
+In today's world, if a Course team wants to change their Course number, they
+have to create a brand new Course with the number they want, despite the actual
+course content being identical to the old Course. Usually, the reason for that
+is so newly created Course Runs under the new Course will have the new Course
+number reflected in their Course Run key and thus LMS URL. When this happens,
```
Comment (mikix) [inline docs/decisions/0006-disallow-course-number-changes-in-publisher.rst]: The `Usually, the reason for that is so newly...` sentence feels awkward to me. Content is fine, just a little hard to read.

File: docs/decisions/0006-disallow-course-number-changes-in-publisher.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
+Disallowing Course Number Changes in Publisher
+==============================================
+
+Status
+------
+
+Accepted
+
+
+Terminology
```
Comment (mikix) [inline docs/decisions/0006-disallow-course-number-changes-in-publisher.rst:10]: Good terminology work.

**Review by mikix — APPROVED** [review]
Yeah seems fine. A little sad, but good to document.

File: docs/decisions/0006-disallow-course-number-changes-in-publisher.rst [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+not allow modifying Course numbers was confirmed after speaking to many of the
+stakeholders involved that could or do use Course keys. Below is documentation
+of the different groups spoken to and the outcomes of those conversations. It
+also includes some avenues that were not explored, but should be if we ever
+decide to revisit this decision.
+
+
+**Marketing** - uses course keys as part of their scripts or reporting. Open to
+switching to UUIDs, but would need to make the switch. See thread here:
+https://edx-internal.slack.com/archives/CD80GQ6NS/p1565716321000400
```
Comment (feanil) [inline docs/decisions/0006-disallow-course-number-changes-in-publisher.rst:93]: Our slack messages are only held for a short time period(less than a year).  I suggest exporting and in-lining any relevant discussions into this document if we want to be able to use it as reference later.

Comment (Dillon-Dumesnil) [reply docs/decisions/0006-disallow-course-number-changes-in-publisher.rst:93]: Good catch! Thanks. I wasn't even thinking about that. I'll fix this up.

**Pull request merged**